components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -409,6 +401,15 @@ - (void)dismissWithAction:(MDCSnackbarMessageAction *)action userInitiated:(BOOL
   }
 }
 
++ (Class)layerClass {
+  return [MDCShadowLayer class];
+}
+
+- (void)setElevation:(MDCShadowElevation)elevation {
+  _elevation = elevation;
+  [(MDCShadowLayer *)self.layer setElevation:_elevation];
+}
+
 #pragma mark - Subclass overrides
```

Comment:
(For Snapshots): Please increase the spacing of the background view around the Snackbar View so the shadow rendering can be fully captured. The easiest way to see if you've fully-captured the shadow is to use the "Digital Color Meter" tool and verify that all 4 edges of the snapshot image are rendered in the background color (0xCCC)

Reply:
Done

Reply:
Done